docs/admin-api-brokers.md

```diff
@@ -14,7 +14,7 @@ import TabItem from '@theme/TabItem';
 
  This page only shows **some frequently used operations**.
 
- - For the latest and complete information about `Pulsar admin`, including commands, flags, descriptions, and more information, see [Pulsar admin doc](/tools/pulsar-admin/).
+ - For the latest and complete information about `Pulsar admin`, including commands, flags, descriptions, and more information, see [Pulsar admin doc](https://pulsar.apache.org/reference/#/@pulsar:version_origin@/pulsar-admin/).
```

Comment:
I'd suggest use `/reference/...` or `pathname:///reference` so that we can locally debug (instead of always link to the online version).

Reply:
Did you mean that we can directly use `pathname://` to replace the domain prefix `https://pulsar.apache.org` for all these occurrences?

Reply:
Said that for references, we bundle all pages: https://github.com/apache/pulsar-site/tree/main/static/reference

Reply:
I've applied this change to more occurrences containing `https://pulsar.apache.org/`, such as `/download/`, `/api/admin/`, `/swagger/` and `/admin-rest-api/`.